### PR TITLE
AMP implementation of Media.net Contextual Monetization

### PR DESCRIPTION
**Pull request by kamlesh-gupta**
Please find the Media.net Contextual Monetization implementation for amp-ad

We have a Google Corporate CLA.
Organization Name - Media.net Advertising FZ-LLC

### CODE REVIEW

**Comment by kamlesh-gupta** [comment]
I signed it!

**Comment by zhouyx** [comment]
This looks good. ping @lannka

File: ads/medianet.md [hunk 1]
```diff
@@ -16,11 +16,26 @@ limitations under the License.
...
+### Media.net Contextual Monetization
+
+``` html
+<amp-ad width="300" height="250"
+    type="medianet"
+    data-tagtype="cm"
+    data-cid="8CUS8O7EX"
+    data-crid="112682482">
+</amp-ad>
+
```
Comment (lannka) [inline ads/medianet.md]: remove  blank lines

File: ads/medianet.md [hunk 2]
```diff
@@ -44,19 +59,33 @@ The ad size depends on the ``width`` and ``height`` attributes specified in the
...
+If you have questions, please feel free to reach out to your Media.net contact.
+
 
 ## Supported Parameters 
 
+###Media.net Contextual Monetization
+
+<strong>Mandatory Parameters</strong>
+ 
+* ``data-tagtype`` - This parameter represents the product the publisher is using; It should be <strong>``cm``</strong> for our <strong>Contextual Monetization solution</strong>
```
Comment (lannka) [inline ads/medianet.md]: why double backquote? Should be single backquote? 
In markdown, you can use `**text**` to make bold text, intead of using `<strong>`.

File: examples/ads.amp.html [hunk 3]
```diff
@@ -796,6 +796,16 @@ <h2 id="medianet">Media.Net Header Bidder Tag</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 >Media.Net Contextual Monetization Tag</h2>
```
Comment (lannka) [inline examples/ads.amp.html]: remove the extra space after `<h2`

**Comment by kamlesh-gupta** [comment]
@lannka - We have made the necessary changes.

**Comment by zhouyx** [comment]
@kamlesh-gupta  Thanks for PR. Looks good. Merging.

**Pull request merged**